tests/rt.conf

```diff
@@ -3,46 +3,35 @@
 
-RUN     | cpld_control_c384                                                                                                       | - wcoss_cray                            | fv3 |
-RUN     | cpld_restart_c384                                                                                                       | - wcoss_cray                            |     | cpld_control_c384
 RUN     | cpld_controlfrac_c384                                                                                                   | - wcoss_cray                            | fv3 |
 RUN     | cpld_restartfrac_c384                                                                                                   | - wcoss_cray                            |     | cpld_controlfrac_c384
 
-RUN     | cpld_bmark                                                                                                              | - wcoss_cray                            | fv3 |
-RUN     | cpld_restart_bmark                                                                                                      | - wcoss_cray                            |     | cpld_bmark
 RUN     | cpld_bmarkfrac                                                                                                          | - wcoss_cray                            | fv3 |
 RUN     | cpld_restart_bmarkfrac                                                                                                  | - wcoss_cray                            |     | cpld_bmarkfrac
 
 #3h/3h/6h restart test
 # test fails on gaea with esmfpio error for tiled output
 RUN     | cpld_bmarkfrac_v16                                                                                                      | - wcoss_cray                            | fv3 |
-RUN     | cpld_bmarkfrac_v16_nsst                                                                                                 | - wcoss_cray                            | fv3 |
 RUN     | cpld_restart_bmarkfrac_v16                                                                                              | - wcoss_cray                            |     | cpld_bmarkfrac_v16
+RUN     | cpld_bmarkfrac_v16_nsst                                                                                                 | - wcoss_cray                            | fv3 |
 
 COMPILE | APP=S2SW SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16_coupled            | - wcoss_cray                            | fv3 |
-RUN     | cpld_bmark_wave                                                                                                         | - wcoss_cray                            | fv3 |
 RUN     | cpld_bmarkfrac_wave                                                                                                     | - wcoss_cray                            | fv3 |
```

Comment:
Do we still need the non-v16 configurations of the benchmarks?

Reply:
That got removed in a second round of cleaning up. I'm not sure why the PR still shows it there.